deploy/Makefile

```diff
@@ -1,32 +1,22 @@
+SHELL := /bin/bash
+
 ARGS=
 
-.PHONY: help bootstrap private network up down clean ps clean-volumes clean-all
+.PHONY: help bootstrap private network up down clean ps clean-volumes clean-all prepare
 
 help:
 	@echo "Usage: make <target>\n"
 	@echo "where <target> is: 'bootstrap', 'up' 'ps', 'down', 'network' or 'clean'\n"
 
 # If DEPLOY_DEV is yes, we use dummy passwords
-bootstrap: ENVS=--env DEPLOY_DEV=yes
-private/cega.yml private/lega.yml private bootstrap:
```

Comment:
I personally don't like the idea of removing `cega.yml`. It's confusing that in case of spinning up local fake-CEGA we have definitions for it in `lega.yml`.